{"user":"x_finder3","timestamp":"2024-07-26T14:54:24.571457","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-26T14:54:24.582380","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-26T14:54:24.592614","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-26T14:54:24.602795","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-26T14:54:24.612997","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-26T14:54:24.623214","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-26T14:54:24.633409","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-26T14:54:24.643598","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-26T14:54:24.653775","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-26T14:54:24.663990","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-26T14:54:24.674285","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-26T14:54:24.684571","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-26T14:54:24.694835","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-26T14:54:24.705060","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-26T14:54:24.715280","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-26T14:54:24.725523","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-26T14:54:24.735735","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-26T14:54:24.745931","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-26T14:54:24.761418","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-26T14:54:24.781428","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-26T14:54:24.801439","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-26T14:54:24.821404","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-26T14:54:24.833786","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-26T14:54:24.843973","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-26T14:54:24.854154","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-26T14:54:24.864313","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-26T14:54:24.874465","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-26T14:54:24.884615","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-26T14:54:24.894781","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-26T14:54:24.904931","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-26T14:54:24.915094","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-26T14:54:24.925285","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-26T14:54:24.935508","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-26T14:54:24.945758","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-26T14:54:24.956003","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-26T14:54:24.966204","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-26T14:54:24.976382","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-26T14:54:24.986571","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-26T14:54:24.996794","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-26T14:54:25.007063","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-26T14:54:25.017303","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-26T14:54:25.041426","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-26T14:54:25.061404","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-26T14:54:25.081450","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-26T14:54:25.101402","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-26T14:54:25.131421","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-26T14:54:25.151412","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-26T14:54:25.171416","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-26T14:54:25.181616","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-26T14:54:25.191826","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-26T14:54:25.202018","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-26T14:54:25.212189","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-26T14:54:25.231487","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-26T14:54:25.241850","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-26T14:54:25.252175","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-26T14:54:25.262500","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-26T14:54:25.273316","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-26T14:54:25.283622","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-26T14:54:25.293921","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-26T14:54:25.304232","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-26T14:54:25.314515","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-26T14:54:25.331423","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-26T14:54:25.351426","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-26T14:54:25.371396","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-26T14:54:26.463332","nav_name":"api_main_json2","action":"启动参数","params":"unix_udp: \/tmp\/main_json,","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-07-26T14:54:26.469420","nav_name":"api_main_json2","action":"组件启动","params":"开始验证授权....","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-07-26T14:54:26.469844","nav_name":"api_main_json2","action":"组件启动","params":"授权验证完成!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-07-26T14:54:26.471083","nav_name":"api_main_json2","action":"组件启动","params":"数据窗口函数-0, 定时任务函数-2","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-07-26T14:54:26.471239","nav_name":"api_main_json2","action":"事件主循环","params":"Unix_UDP 建立中","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-07-26T14:54:26.471401","nav_name":"api_main_json2","action":"启动完成","params":"Unix_UDP 准备完毕,开始接收消息!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-07-26T15:00:00.626314","nav_name":"api_main_json2","action":"启动参数","params":"unix_udp: \/tmp\/main_json,","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-07-26T15:00:00.647967","nav_name":"api_main_json2","action":"组件启动","params":"开始验证授权....","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-07-26T15:00:00.648344","nav_name":"api_main_json2","action":"组件启动","params":"授权验证完成!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-07-26T15:00:00.649843","nav_name":"api_main_json2","action":"组件启动","params":"数据窗口函数-0, 定时任务函数-2","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-07-26T15:00:00.650000","nav_name":"api_main_json2","action":"事件主循环","params":"Unix_UDP 建立中","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-07-26T15:00:00.650144","nav_name":"api_main_json2","action":"启动完成","params":"Unix_UDP 准备完毕,开始接收消息!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-07-26T15:06:07.785046","nav_name":"api_main_json2","action":"启动参数","params":"unix_udp: \/tmp\/main_json,","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-07-26T15:06:07.796723","nav_name":"api_main_json2","action":"组件启动","params":"开始验证授权....","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-07-26T15:06:07.797128","nav_name":"api_main_json2","action":"组件启动","params":"授权验证完成!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-07-26T15:06:07.798487","nav_name":"api_main_json2","action":"组件启动","params":"数据窗口函数-0, 定时任务函数-2","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-07-26T15:06:07.798761","nav_name":"api_main_json2","action":"事件主循环","params":"Unix_UDP 建立中","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-07-26T15:06:07.798932","nav_name":"api_main_json2","action":"启动完成","params":"Unix_UDP 准备完毕,开始接收消息!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-07-26T16:43:07.088973","nav_name":"api_main_json2","action":"启动参数","params":"unix_udp: \/tmp\/main_json,","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-07-26T16:43:07.094802","nav_name":"api_main_json2","action":"组件启动","params":"开始验证授权....","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-07-26T16:43:07.095194","nav_name":"api_main_json2","action":"组件启动","params":"授权验证完成!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-07-26T16:43:07.096134","nav_name":"api_main_json2","action":"组件启动","params":"数据窗口函数-0, 定时任务函数-2","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-07-26T16:43:07.096288","nav_name":"api_main_json2","action":"事件主循环","params":"Unix_UDP 建立中","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-07-26T16:43:07.096455","nav_name":"api_main_json2","action":"启动完成","params":"Unix_UDP 准备完毕,开始接收消息!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-07-29T11:02:17.360700","nav_name":"api_main_json2","action":"启动参数","params":"unix_udp: \/tmp\/main_json,","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-07-29T11:02:17.367096","nav_name":"api_main_json2","action":"组件启动","params":"开始验证授权....","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-07-29T11:02:17.367489","nav_name":"api_main_json2","action":"组件启动","params":"授权验证完成!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-07-29T11:02:17.368450","nav_name":"api_main_json2","action":"组件启动","params":"数据窗口函数-0, 定时任务函数-2","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-07-29T11:02:17.368602","nav_name":"api_main_json2","action":"事件主循环","params":"Unix_UDP 建立中","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-07-29T11:02:17.368748","nav_name":"api_main_json2","action":"启动完成","params":"Unix_UDP 准备完毕,开始接收消息!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-07-29T11:05:39.158429","nav_name":"api_main_json2","action":"启动参数","params":"unix_udp: \/tmp\/main_json,","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-07-29T11:05:39.200858","nav_name":"api_main_json2","action":"组件启动","params":"开始验证授权....","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-07-29T11:05:39.201242","nav_name":"api_main_json2","action":"组件启动","params":"授权验证完成!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-07-29T11:05:39.202895","nav_name":"api_main_json2","action":"组件启动","params":"数据窗口函数-0, 定时任务函数-2","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-07-29T11:05:39.203098","nav_name":"api_main_json2","action":"事件主循环","params":"Unix_UDP 建立中","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-07-29T11:05:39.203281","nav_name":"api_main_json2","action":"启动完成","params":"Unix_UDP 准备完毕,开始接收消息!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:02.931512","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:02.942029","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:02.952279","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:02.962491","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:02.977532","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:02.987833","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:02.998058","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:03.008275","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:03.018490","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:03.028719","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:03.038925","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:03.049177","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:03.059508","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:03.069846","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:03.080163","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:03.090478","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:03.100697","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:03.110921","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:03.121155","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:03.131396","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:03.141600","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:03.151793","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:03.161988","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:03.172187","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:03.182386","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:03.192566","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:03.202750","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:03.212932","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:03.241510","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:03.261440","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:03.271728","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:03.281994","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:03.292244","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:03.302483","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:03.312725","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:03.322964","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:03.333219","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:03.343492","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:03.353708","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:03.363912","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:03.391421","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:03.411428","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:03.421686","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:03.431938","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:03.442206","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:03.461622","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:03.481745","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:03.501946","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:03.531435","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:03.541703","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:03.551944","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:03.562186","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:03.572411","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:03.582645","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:03.592866","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:03.603086","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:03.613302","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:03.623525","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:03.633746","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:03.643981","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:03.654217","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:03.664446","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:03.681491","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:03.701616","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:03.721421","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:03.731594","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:03.751392","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:03.771398","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:03.791403","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:03.801597","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:03.811805","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:03.821975","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:03.832159","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:03.842343","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:03.852528","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:03.862711","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:03.872902","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:03.883089","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:03.893266","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:03.903440","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:03.921412","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:03.941411","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:03.961407","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:03.981409","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:04.001427","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:04.021488","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:04.031760","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:04.042016","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:04.052249","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:04.062459","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:04.072677","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:04.082916","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:04.093181","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:04.103454","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:04.113735","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:05.325018","nav_name":"api_main_json2","action":"启动参数","params":"unix_udp: \/tmp\/main_json,","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:05.330381","nav_name":"api_main_json2","action":"组件启动","params":"开始验证授权....","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:05.330750","nav_name":"api_main_json2","action":"组件启动","params":"授权验证完成!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:05.331784","nav_name":"api_main_json2","action":"组件启动","params":"数据窗口函数-0, 定时任务函数-2","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:05.331943","nav_name":"api_main_json2","action":"事件主循环","params":"Unix_UDP 建立中","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-07-30T09:38:05.332110","nav_name":"api_main_json2","action":"启动完成","params":"Unix_UDP 准备完毕,开始接收消息!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-07-30T09:49:57.143674","nav_name":"api_main_json2","action":"启动参数","params":"unix_udp: \/tmp\/main_json,","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-07-30T09:49:57.149742","nav_name":"api_main_json2","action":"组件启动","params":"开始验证授权....","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-07-30T09:49:57.150139","nav_name":"api_main_json2","action":"组件启动","params":"授权验证完成!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-07-30T09:49:57.151346","nav_name":"api_main_json2","action":"组件启动","params":"数据窗口函数-0, 定时任务函数-2","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-07-30T09:49:57.151532","nav_name":"api_main_json2","action":"事件主循环","params":"Unix_UDP 建立中","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-07-30T09:49:57.151700","nav_name":"api_main_json2","action":"启动完成","params":"Unix_UDP 准备完毕,开始接收消息!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-07-30T09:53:48.285009","nav_name":"api_main_json2","action":"启动参数","params":"unix_udp: \/tmp\/main_json,","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-07-30T09:53:48.291090","nav_name":"api_main_json2","action":"组件启动","params":"开始验证授权....","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-07-30T09:53:48.291494","nav_name":"api_main_json2","action":"组件启动","params":"授权验证完成!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-07-30T09:53:48.292695","nav_name":"api_main_json2","action":"组件启动","params":"数据窗口函数-0, 定时任务函数-2","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-07-30T09:53:48.292891","nav_name":"api_main_json2","action":"事件主循环","params":"Unix_UDP 建立中","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-07-30T09:53:48.293047","nav_name":"api_main_json2","action":"启动完成","params":"Unix_UDP 准备完毕,开始接收消息!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-07-30T10:04:09.642103","nav_name":"api_main_json2","action":"启动参数","params":"unix_udp: \/tmp\/main_json,","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-07-30T10:04:09.648541","nav_name":"api_main_json2","action":"组件启动","params":"开始验证授权....","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-07-30T10:04:09.648908","nav_name":"api_main_json2","action":"组件启动","params":"授权验证完成!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-07-30T10:04:09.650223","nav_name":"api_main_json2","action":"组件启动","params":"数据窗口函数-0, 定时任务函数-2","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-07-30T10:04:09.650373","nav_name":"api_main_json2","action":"事件主循环","params":"Unix_UDP 建立中","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-07-30T10:04:09.650515","nav_name":"api_main_json2","action":"启动完成","params":"Unix_UDP 准备完毕,开始接收消息!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-07-30T10:12:16.037619","nav_name":"api_main_json2","action":"启动参数","params":"unix_udp: \/tmp\/main_json,","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-07-30T10:12:16.057305","nav_name":"api_main_json2","action":"组件启动","params":"开始验证授权....","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-07-30T10:12:16.057712","nav_name":"api_main_json2","action":"组件启动","params":"授权验证完成!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-07-30T10:12:16.059315","nav_name":"api_main_json2","action":"组件启动","params":"数据窗口函数-0, 定时任务函数-2","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-07-30T10:12:16.059523","nav_name":"api_main_json2","action":"事件主循环","params":"Unix_UDP 建立中","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-07-30T10:12:16.059707","nav_name":"api_main_json2","action":"启动完成","params":"Unix_UDP 准备完毕,开始接收消息!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-07-30T10:17:50.720501","nav_name":"api_main_json2","action":"启动参数","params":"unix_udp: \/tmp\/main_json,","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-07-30T10:17:50.727180","nav_name":"api_main_json2","action":"组件启动","params":"开始验证授权....","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-07-30T10:17:50.727575","nav_name":"api_main_json2","action":"组件启动","params":"授权验证完成!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-07-30T10:17:50.729509","nav_name":"api_main_json2","action":"组件启动","params":"数据窗口函数-0, 定时任务函数-2","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-07-30T10:17:50.729896","nav_name":"api_main_json2","action":"事件主循环","params":"Unix_UDP 建立中","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-07-30T10:17:50.730176","nav_name":"api_main_json2","action":"启动完成","params":"Unix_UDP 准备完毕,开始接收消息!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-07-31T16:17:52.701038","nav_name":"api_main_json2","action":"启动参数","params":"unix_udp: \/tmp\/main_json,","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-07-31T16:17:52.706480","nav_name":"api_main_json2","action":"组件启动","params":"开始验证授权....","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-07-31T16:17:52.706851","nav_name":"api_main_json2","action":"组件启动","params":"授权验证完成!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-07-31T16:17:52.707864","nav_name":"api_main_json2","action":"组件启动","params":"数据窗口函数-0, 定时任务函数-2","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-07-31T16:17:52.708020","nav_name":"api_main_json2","action":"事件主循环","params":"Unix_UDP 建立中","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-07-31T16:17:52.708182","nav_name":"api_main_json2","action":"启动完成","params":"Unix_UDP 准备完毕,开始接收消息!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-08-01T17:38:51.096215","nav_name":"api_main_json2","action":"启动参数","params":"unix_udp: \/tmp\/main_json,","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-08-01T17:38:51.102709","nav_name":"api_main_json2","action":"组件启动","params":"开始验证授权....","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-08-01T17:38:51.103060","nav_name":"api_main_json2","action":"组件启动","params":"授权验证完成!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-08-01T17:38:51.104286","nav_name":"api_main_json2","action":"组件启动","params":"数据窗口函数-0, 定时任务函数-2","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-08-01T17:38:51.104467","nav_name":"api_main_json2","action":"事件主循环","params":"Unix_UDP 建立中","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-08-01T17:38:51.104616","nav_name":"api_main_json2","action":"启动完成","params":"Unix_UDP 准备完毕,开始接收消息!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-08-01T17:39:51.055980","nav_name":"api_main_json2","action":"启动参数","params":"unix_udp: \/tmp\/main_json,","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-08-01T17:39:51.070642","nav_name":"api_main_json2","action":"组件启动","params":"开始验证授权....","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-08-01T17:39:51.071611","nav_name":"api_main_json2","action":"组件启动","params":"授权验证完成!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-08-01T17:39:51.074087","nav_name":"api_main_json2","action":"组件启动","params":"数据窗口函数-0, 定时任务函数-2","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-08-01T17:39:51.074330","nav_name":"api_main_json2","action":"事件主循环","params":"Unix_UDP 建立中","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-08-01T17:39:51.074592","nav_name":"api_main_json2","action":"启动完成","params":"Unix_UDP 准备完毕,开始接收消息!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-08-06T09:34:07.726795","nav_name":"api_main_json2","action":"启动参数","params":"unix_udp: \/tmp\/main_json,","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-08-06T09:34:07.742925","nav_name":"api_main_json2","action":"组件启动","params":"开始验证授权....","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-08-06T09:34:07.743282","nav_name":"api_main_json2","action":"组件启动","params":"授权验证完成!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-08-06T09:34:07.744228","nav_name":"api_main_json2","action":"组件启动","params":"数据窗口函数-0, 定时任务函数-2","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-08-06T09:34:07.744382","nav_name":"api_main_json2","action":"事件主循环","params":"Unix_UDP 建立中","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-08-06T09:34:07.744540","nav_name":"api_main_json2","action":"启动完成","params":"Unix_UDP 准备完毕,开始接收消息!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-08-06T09:40:40.511536","nav_name":"api_main_json2","action":"启动参数","params":"unix_udp: \/tmp\/main_json,","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-08-06T09:40:40.517172","nav_name":"api_main_json2","action":"组件启动","params":"开始验证授权....","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-08-06T09:40:40.517561","nav_name":"api_main_json2","action":"组件启动","params":"授权验证完成!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-08-06T09:40:40.518623","nav_name":"api_main_json2","action":"组件启动","params":"数据窗口函数-0, 定时任务函数-2","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-08-06T09:40:40.518779","nav_name":"api_main_json2","action":"事件主循环","params":"Unix_UDP 建立中","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-08-06T09:40:40.518948","nav_name":"api_main_json2","action":"启动完成","params":"Unix_UDP 准备完毕,开始接收消息!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-08-06T09:42:00.502966","nav_name":"api_main_json2","action":"启动参数","params":"unix_udp: \/tmp\/main_json,","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-08-06T09:42:00.509320","nav_name":"api_main_json2","action":"组件启动","params":"开始验证授权....","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-08-06T09:42:00.509734","nav_name":"api_main_json2","action":"组件启动","params":"授权验证完成!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-08-06T09:42:00.511184","nav_name":"api_main_json2","action":"组件启动","params":"数据窗口函数-0, 定时任务函数-2","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-08-06T09:42:00.511394","nav_name":"api_main_json2","action":"事件主循环","params":"Unix_UDP 建立中","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-08-06T09:42:00.511580","nav_name":"api_main_json2","action":"启动完成","params":"Unix_UDP 准备完毕,开始接收消息!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:03.087841","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:03.098334","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:03.108622","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:03.118847","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:03.137675","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:03.147966","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:03.158225","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:03.177718","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:03.197736","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:03.208041","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:03.218330","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:03.237748","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:03.257717","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:03.277705","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:03.297724","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:03.317698","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:03.337693","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:03.357697","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:03.377698","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:03.389381","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:03.399640","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:03.409928","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:03.420236","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:03.430568","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:03.440909","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:03.451217","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:03.461511","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:03.471771","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:03.482004","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:03.492265","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:03.502579","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:03.512948","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:03.523275","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:03.533596","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:03.543961","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:03.554363","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:03.577706","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:03.588031","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:03.598236","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:03.617664","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:03.637669","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:03.647955","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:03.658245","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:03.668516","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:03.678749","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:03.697684","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:03.707982","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:03.718247","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:03.737700","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:03.748038","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:03.758342","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:03.777688","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:03.797680","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:03.807965","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:03.818180","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:03.828387","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:03.838635","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:03.848893","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:03.868518","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:03.887730","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:03.898016","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:03.917753","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:03.928099","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:03.938436","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:03.948785","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:03.959124","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:03.969453","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:03.987747","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:03.998090","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:04.017733","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:04.037725","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:04.057730","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:04.077705","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:04.087970","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:04.098197","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:04.108412","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:04.118607","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:04.128802","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:04.139001","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:04.149190","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:04.159374","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:04.169564","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:04.179753","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:04.189942","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:04.200132","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:04.210329","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:04.220526","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:04.230720","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:04.240923","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:04.251153","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:04.261399","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:04.271624","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:04.281804","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:04.291979","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:04.302148","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:04.312334","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:05.538399","nav_name":"api_main_json2","action":"启动参数","params":"unix_udp: \/tmp\/main_json,","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:05.543800","nav_name":"api_main_json2","action":"组件启动","params":"开始验证授权....","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:05.544185","nav_name":"api_main_json2","action":"组件启动","params":"授权验证完成!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:05.545154","nav_name":"api_main_json2","action":"组件启动","params":"数据窗口函数-0, 定时任务函数-2","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:05.545305","nav_name":"api_main_json2","action":"事件主循环","params":"Unix_UDP 建立中","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-08-09T16:58:05.545466","nav_name":"api_main_json2","action":"启动完成","params":"Unix_UDP 准备完毕,开始接收消息!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-08-15T15:11:06.475298","nav_name":"api_main_json2","action":"启动参数","params":"unix_udp: \/tmp\/main_json,","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-08-15T15:11:06.480602","nav_name":"api_main_json2","action":"组件启动","params":"开始验证授权....","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-08-15T15:11:06.480937","nav_name":"api_main_json2","action":"组件启动","params":"授权验证完成!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-08-15T15:11:06.482138","nav_name":"api_main_json2","action":"组件启动","params":"数据窗口函数-0, 定时任务函数-2","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-08-15T15:11:06.482286","nav_name":"api_main_json2","action":"事件主循环","params":"Unix_UDP 建立中","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-08-15T15:11:06.482424","nav_name":"api_main_json2","action":"启动完成","params":"Unix_UDP 准备完毕,开始接收消息!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-08-27T16:27:55.237729","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-27T16:27:55.248881","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-27T16:27:55.259166","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-27T16:27:55.269453","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-27T16:27:55.288212","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-27T16:27:55.307798","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-27T16:27:55.327687","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-27T16:27:55.347844","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-27T16:27:55.362092","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-27T16:27:55.377689","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-27T16:27:55.397686","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-27T16:27:55.417796","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-27T16:27:55.428043","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-27T16:27:55.438258","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-27T16:27:55.448476","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-27T16:27:55.458701","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-27T16:27:55.468931","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-27T16:27:55.479168","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-27T16:27:55.497739","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-27T16:27:55.517730","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-27T16:27:55.537732","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-27T16:27:55.547968","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-27T16:27:55.564226","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-27T16:27:55.574431","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-27T16:27:55.584632","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-27T16:27:55.594830","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-27T16:27:55.605028","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-27T16:27:55.615228","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-27T16:27:55.625438","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-27T16:27:55.635649","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-27T16:27:55.645864","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-27T16:27:55.656083","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-27T16:27:55.666288","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-27T16:27:55.676500","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-27T16:27:55.686712","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-27T16:27:55.696919","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-27T16:27:55.707124","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-27T16:27:55.717372","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-27T16:27:55.727647","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-27T16:27:55.737889","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-27T16:27:55.748105","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-27T16:27:55.758298","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-27T16:27:55.768482","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-27T16:27:55.778676","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-27T16:27:55.788869","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-27T16:27:55.799056","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-27T16:27:55.817674","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-27T16:27:55.837646","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-27T16:27:55.847840","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-27T16:27:55.858026","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-27T16:27:55.877661","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-27T16:27:55.888919","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-27T16:27:55.899176","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-27T16:27:55.909429","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-27T16:27:55.919691","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-27T16:27:55.929936","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-27T16:27:55.940174","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-27T16:27:55.950416","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-27T16:27:55.962345","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-27T16:27:55.977748","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-27T16:27:55.987982","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-27T16:27:55.998187","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-27T16:27:56.008385","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-27T16:27:56.018578","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-27T16:27:56.028790","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-27T16:27:56.039025","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-27T16:27:56.049245","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-27T16:27:56.059455","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-27T16:27:56.072653","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-27T16:27:56.087754","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-27T16:27:56.107749","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-27T16:27:56.127724","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-27T16:27:56.147730","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-27T16:27:56.157932","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-27T16:27:56.168127","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-27T16:27:56.178326","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-27T16:27:56.188511","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-27T16:27:56.198697","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-27T16:27:56.208878","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-27T16:27:56.219069","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-27T16:27:56.229261","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-27T16:27:56.239456","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-27T16:27:56.257791","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-27T16:27:56.277667","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-27T16:27:56.287862","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-27T16:27:56.298072","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-27T16:27:56.317727","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-27T16:27:56.327976","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-27T16:27:56.338219","nav_name":"api_main_json2","action":"发送消息","params":"Unix_udp \/tmp\/api_object","operate_result":"错误","failed_reason":"[Errno 111] Connection refused"}
{"user":"x_finder3","timestamp":"2024-08-27T16:27:57.639507","nav_name":"api_main_json2","action":"启动参数","params":"unix_udp: \/tmp\/main_json,","operate_result":"信息","failed_reason":""}
